CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 ## HEAD (Unreleased)
 
+---
+
+## 6.1.0 (2024-12-19)
```

Comment:
Should it be 6.0.1? Also the date will need to be changed to the actual date we release it. Might be worth holding off on indicating a release until we're about to do one. I have another change I want to release as well: https://github.com/pulumi/actions/pull/1327